client/indexer-db/src/models/peer_id.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{schema::peer_id, DbConnection};
+
+/// Table that holds the list of peers that the indexer is interested in.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = peer_id)]
+pub struct PeerId {
+    pub id: i32,
+    pub peer: Vec<u8>,
+    pub created_at: NaiveDateTime,
+    pub updated_at: NaiveDateTime,
```

Comment:
Why a separate table for this? I

Reply:
Just the way relational databases best handle arrays. Otherwise would be hard (if not impossible) to query for example by `PeerId`.